doc/api.md

```diff
@@ -1,122 +1,143 @@
 # IoT Agent API
 
-<!-- TOC WIP -->
+<!-- TOC -->
+
+-   [Preface](#preface)
+-   [Topics](#topics)
+    -   [Terminology](#terminology)
+    -   [IoT Agent information model](#iot-agent-information-model)
+        -   [Config groups](#config-groups)
+        -   [Devices](#devices)
+    -   [Entity attributes](#entity-attributes)
+    -   [Advice on Attribute definitions](#advice-on-attribute-definitions)
+        -   [Reuse of attributes names](#reuse-of-attributes-names)
```

Comment:
Link seems to be broken.

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1336/commits/7b5379132d70466470812d65f22f04fc594e526b